packages/server/postgres/migrations/1652861495447_addIntegrationSearchQuery.ts

```diff
@@ -0,0 +1,44 @@
+import {Client} from 'pg'
+import getPgConfig from '../getPgConfig'
+
+export async function up() {
+  const client = new Client(getPgConfig())
+  await client.connect()
+  await client.query(`
+    CREATE TABLE IF NOT EXISTS "IntegrationSearchQuery" (
+      "id" INT GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
+      "userId" VARCHAR(100) NOT NULL,
+      "teamId" VARCHAR(100) NOT NULL,
+      "providerId" INT,
+      "service" "IntegrationProviderServiceEnum" NOT NULL,
+      "query" JSONB NOT NULL DEFAULT '{}',
+      "lastUsedAt" TIMESTAMP WITH TIME ZONE DEFAULT CURRENT_TIMESTAMP NOT NULL,
+      "createdAt" TIMESTAMP WITH TIME ZONE DEFAULT CURRENT_TIMESTAMP NOT NULL,
+      "updatedAt" TIMESTAMP WITH TIME ZONE DEFAULT CURRENT_TIMESTAMP NOT NULL,
```

Comment:
-1 if we add an "updatedAt" field we should also add a trigger to automatically set it https://github.com/ParabolInc/parabol/pull/6493